pages/guides/how-to-schedule-price-updates-with-gelato.mdx

```diff
@@ -0,0 +1,125 @@
+
+Once the task is running, you can update this configuration to add more price feeds or change the update parameters.
+The task will automatically pick up configuration changes.
+
+#### 3. Create the task
+
+Use the link below to auto-populate the task parameters with the Pyth web3 function code:
+
+[//]: # "TODO: check/update this link"
+
+https://beta.app.gelato.network/new-task?cid=QmTFQHpiThqkVKvuW2KU8cETNBdyXHB9uP34woL6Vc5w6g
+
+The pyth web3 function code is deployed to IPFS, so you can use it via the cid/link above.
+you can find a copy of the web3 function code [here](https://github.com/gelatodigital/w3f-pyth-poc-v2/tree/master/web3-functions/pyth-oracle-w3f-priceIds)
+
+Choose network, and then in the "Task secrets" section, set `GIST_ID` to the gistId of the gist you created in step 2.
+![](../../images/guides/gelato/task-creation-gist-id.png)
+
+** Note that the beta currently only supports triggering the task every minute. Once other time intervals are available,
+you will need to ensure that your current subscription plan with Gelato supports the desired number of monthly runs.
+See (Subscription Plans)[https://beta.app.gelato.network/subscriptions] **.
+
+#### 4. Fund the task
+
+There are two sets of fees that must be paid before the task can execute successfully
+
+**Gelato Fees**
+Gelato's fees are paid through [1Balance](https://docs.gelato.network/developer-services/1balance). Once you are whitelisted, visit the [one balance app](https://beta.app.gelato.network/balance)
+and deposit USDC on Polygon to top up your Gelato balance which will be used to pay the Gelato fees on all
+supported chains. The Testnet executions are subsidized by Gelato and free. Make sure to create your 1Balance account
+using the same wallet address that you used to create the Gelato task.
+
+Switch network to Polygon
+![](../../images/guides/gelato/switch-to-polygon.png)
+
+Deposit USDC
+![](../../images/guides/gelato/deposit-usdc.png)
+
+**Pyth Fees**
+These are the fees that will be paid out to the Pyth oracle. The Pyth fees are paid in the native token of the chain
+where the task is running. The method that updates the price is payable, the update transaction has to include in the
+`msg.value` the corresponding fee.
+
+The on-chain transaction executed via a web3 function gets routed through a proxy smart contract
+which is solely owned by the web3 function task creator. This proxy smart contract is known as the `dedicatedMsgSender`
+and will be deployed the first time a task is created for a given wallet address.
+
+The `dedicatedMsgSender` is also responsible for paying the Pyth Fees. You will need to transfer over the native tokens
+to the `dedicatedMsgSender` on every chain where you want to run the task using a standard transfer.
+The `dedicatedMsgSender` address can be found in the Gelato task details under `Target smart contract` as the
+`Your msg.sender` field
+
+![](../../images/guides/gelato/dedicated-msg-sender.png)
+
```

Comment:
Suggest being explicit here about how to do this transfer. "You can simply open your wallet and send native tokens to this address"